src/applications/disability-benefits/526EZ/components/AsyncDisplayWidget.jsx

```diff
@@ -43,7 +43,8 @@ export default class AsyncDisplayWidget extends React.Component {
   componentDidMount() {
     // TODO: Don't call the callback _every_ time the component is mounted
     const cbPromise = this.props.options.callback();
-    if (cbPromise instanceof Promise) {
+    // instanceof Promise doesn't work in Firefox, so we just check for .then() and hope it's a promise
+    if (typeof cbPromise.then === 'function') {
```

Comment:
You might want to check that cbPromise is truthy, otherwise if that callback function returns undefined or something, you're going to throw a different error than the one in the else block.